features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Endpoint/ApiCreateEndpoint.js

```diff
@@ -72,6 +72,10 @@ class ApiCreateEndpoint extends React.Component {
         apiFields["selectedPolicies"] = policies;
         this.setState({apiFields:apiFields});
     }
+    componentWillMount(){
+        //Do the call to service discovery endpoint
+        console.info('print the response json');
```

Comment:
Is this not implemented yet? if so please add a comment and a TODO comment.  and remove the console logs if not needed

Reply:
It was a part of a previous plan. Thanks, I forgot it was there. I'm removing it.